### PR TITLE
Update software page

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated the markup of the /programmes/software page

## QA

- `./run`
- Go to /programmes/software

## Screenshots

image

### CODE REVIEW

**Comment by kwm14** [comment]
LGTM 👍

File: templates/programmes/software.html [hunk 1]
```diff
@@ -4,82 +4,77 @@
...
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h1 class="p-heading--two">Ubuntu software partners</h1>
+      <p>Ubuntu has a thriving software ecosystem, with thousands of applications available for the PC, the server and the cloud &mdash; targeting both the enterprise and consumer markets. </p>
+      <p>We offer a range of benefits for ISVs looking to formalise their work with Ubuntu, through Canonical. As a certified partner, you&rsquo;ll get inside information on forthcoming Ubuntu developments, assistance in testing and optimising your applications with Ubuntu and OpenStack &mdash; and priority access to Ubuntu engineers.</p>
+      <p><a href="/contact-us">Talk to us about becoming a partner&nbsp;&rsaquo;</a></p>
     </div>
-    <img class="priority-0" src="https://assets.ubuntu.com/v1/a3d5299d-image-scopes.jpg" alt="scopes">
+    <div class="col-4">
+      <img class="priority-0" src="https://assets.ubuntu.com/v1/a3d5299d-image-scopes.jpg" alt="scopes">
```
Comment (anthonydillon) [inline templates/programmes/software.html]: s/priority-0/u-hide--small

File: templates/programmes/software.html [hunk 2]
```diff
@@ -4,82 +4,77 @@
...
-        </div>
-        <div class="four-col last-col align-center align-vertically">
-            <img class="priority-0" width="150" src="https://assets.ubuntu.com/v1/527fbcef-picto-developer-warmgrey.svg" alt="developer pictogram">
-        </div>
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h2 class="p-heading--three">App developer partners</h2>
+      <p>Thanks to the ease with which users can find and install software using the Ubuntu Software Centre, Ubuntu already supports thousands of client applications. With the launch of Ubuntu phones and tablets &mdash; and the resulting opportunity to target three client platforms with just one codebase &mdash; the Ubuntu app market is set to grow exponentially. </p>
+      <p><a href="https://developer.ubuntu.com/en/start/ubuntu-sdk/installing-the-sdk/" class="external">Learn more about Ubuntu SDK</a></p>
```
Comment (anthonydillon) [inline templates/programmes/software.html]: `p-link--external`

**Comment by deadlight** [comment]
@anthonydillon fixed up the branch

**Pull request merged**